cmd/snap/cmd_debug_api.go

```diff
@@ -0,0 +1,108 @@
+// -*- Mode: Go; indent-tabs-mode: t -*-
```

Comment:
I think somewhere here it warrants a simple example just to show how it's used

Reply:
If you add an example (like you have in the PR description), you could add it to the command's long description so it's easily accessible

Reply:
I've added some examples in long description of the command.